types/usb/index.d.ts

```diff
@@ -60,7 +61,7 @@ export class Interface {
   endpoints: Endpoint[];
   constructor(device: Device, id: number);
   claim(): void;
-  release(closeEndpoints?: (err?: string) => void, cb?: (err?: string) => void): void;
+  release(closeEndpoints?: boolean, cb?: (err?: string) => void): void;
```

Comment:
This is not the correct implementation of `closeEndpoints`, if you look at https://github.com/tessel/node-usb/blob/master/usb.js#L187 it can optionally be a `Function`, so we should make this support both

Reply:
Both correct implementations now exist